### DISCUSSION TITLE
WebDav problem： Icon preview disappear in Finder

### DISCUSSION THREAD

**Discussion by will-huang9531** [iterate-ch/cyberduck#15511] [Q&A]
<img width="1183" alt="iShot_2024-01-05_09 11 01" src="https://github.com/iterate-ch/cyberduck/assets/103096674/24494e41-0421-43bb-8ef7-dfc08ff21193">

macOS version：14.2.1
mountain duck version：4.15.1 (21679)
When I mount a network disk using the webdav, the Icon preview doesn't work in finder, so I cannot see the thumbnails.
I'm very confident that “Finder → View → Show View Options → Show Icon Preview” has been enabled.
Same problem as: WebDav problem：No thumbnail preview #15454

**Comment by dkocher** [comment]
Duplicate for #15454.
